docs/source/2019Barry.rst

```diff
@@ -72,6 +72,32 @@ Model Configuration and Datasets
    
       crontab Barry.crontab  
 
+  .. group-tab:: SRW.v1.0
+
+    The UFS Short-Range Weather (SRW) Application (App) is used to prepare initial conditions, compile and run the UFS model, and post process the raw model outputs. Two model configuration compsets (``GFSv15p2`` and ``RRFSv1alpha``) are tested using the :emphasis:`C768` (~13km) spatial resolution with 64 vertical levels (default).
```

Comment:
I would use "post-process" or "postprocess", both versions are more commonly used than "post process"

Reply:
"postprocess" is used, thanks!